tests/starship/configs/ci.yaml

```diff
@@ -0,0 +1,29 @@
+chains:
+  - name: simapp
+    type: simapp
+    image: ghcr.io/cosmology-tech/starship/simapp:v0.47.3
```

Comment:
ditto

Reply:
I can change this to build from branch. Would that make sense?

Reply:
We can then basically, have a build step, to build and push a docker image. Then we use that to run the tests.

Reply:
Let me figure out the best way, that will use the least amount of time.